tests/mve/gtest_view.cc

```diff
@@ -8,176 +8,179 @@
 
```

Comment:
The only real change in this file is the addition on line 12. Everything else is just replacement of a View with View::Ptr.